lib/matplotlib/widgets.py

```diff
@@ -4199,19 +4199,26 @@ class Lasso(AxesWidget):
         Whether to use blitting for faster drawing (if supported by the
         backend). See the tutorial :ref:`blitting`
         for details.
+    props: dict, optional
+        Lasso line properties. See `matplotlib.lines.Line2D` for valid
```

Comment:
```suggestion
        Lasso line properties. See `.Line2D` for valid
```
This will give a shorter link text (someone should change for LassoSelector as well...).

Reply:
Thanks oscargus! Added